Eastern-Research-Group/expert-query
# Feature/111 remove proxy code

## Related Issues:
* [EQ-111](https://jira.epa.gov/browse/EQ-111)

## Main Changes:
* Removed the `proxy` endpoint from the app server.
* Removed the `profile_test` table from the etl process.
* Removed the `data/profileTests` and `data/profileTests/count` endpoints.
* Removed the `api/test` endpoint.
* Removed other unnecessary code.
* Updated the status being returned when the `400.html` and `500.html` files are returned from the app server.

## Steps To Test:
1. Run the etl and verify the `profile_test` table is not created.
2. Run the app
3. Verify the proxy endpoint can't be reached: localhost:9090?proxy=https://google.com
4. Verify the `data/profileTests` and `data/profileTests/count` endpoints can't be reached
5. Verify the `api/test` endpoint can't be reached.
